site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,149 @@
+
+{% instruction 'Update our `main.ts` file to import our theme and create a `themeInjector`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' %}
+
+Open the application in your web browser and see that the `Worker` backgrounds are now `red`.
+
+{% instruction 'Use variables and complete the Worker theme' %}
+
+The [Dojo 2 build system](../006_deploying_to_production/) supports new CSS features such as `css-custom-properties` by using PostCSS to process our `.m.css` files. We can use these new CSS features to add variables to `worker.m.css` and complete its theme.
+Let's create `themes/dojo/variables.css` (notice that this file does not have a `.m.css` extension as it is not a `css-module` file).
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/variables.css' lang:css %}
+
+We can now use these variables in `worker.m.css` to create our fully themed `Worker`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/worker.m.css' lang:css %}
+
+{% section %}
+
+## Theming Dojo widgets
+
+Thus far in this tutorial, we have themed our custom `Worker` widget, but we have not targeted Dojo widgets that are contained within our application? To demonstrate the styling of Dojo widgets, we will theme the `workerForm` widget as it contains both DOM nodes and Dojo widgets.
+
+{% instruction 'Let\'s create `workerForm.m.css` and include it in theme.ts' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/workerForm.m.css' lang:css %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,5,9,11,13,16 %}
+
+This should be familiar from theming the `Worker` in the previous section. To theme the Dojo 2 `TextInput` within our `WorkerForm`, we need to create `dojoTextInput.m.css` and export it from `theme.ts` using a theme key prefixed with `dojo-`. The dojo prefix helps ensure that the Dojo widget theme keys do not clash with application widget theme keys.
+
+{% instruction 'Create `dojoTextInput.m.css` and export it from `theme.ts`' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/dojoTextInput.m.css' lang:css %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,5,6,9,11,13,14,16 %}
+
+In this example we are introducing another powerful part of the Dojo 2 theming system, `composes`. Composes originates in the `css-module` specification, allowing you to apply styles from one class selector to another. Here we are specifying that `root` of a `TextInput` (the label text in this case), should appear the same as the `nameLabel` class in our `WidgetForm`. This approach can be very useful when creating multiple themes from a `baseTheme` and avoids repetitive redefinition of style rules.
```

Comment:
Please callout where `compose` is being used, in case people are copy-and-pasting (like I was...). For example, "Notice the styling rule for the `.root` selector? This demonstrates another powerful part..."